python/lsst/sims/maf/metricBundles/moMetricBundle.py

```diff
@@ -72,6 +72,8 @@ def makeCompletenessBundle(bundle, completenessMetric, Hmark=None, resultsDb=Non
         mb.setSummaryMetrics(metric)
         mb.computeSummaryStats(resultsDb)
         val = mb.summaryValues['Value At H=%.1f' % Hmark]
+        if val is None:
+            val = 0
```

Comment:
I fixed this another way somewhere .. must be from the run script? Maybe I need to move that fix further down. Basically, setting the val to 0 is not a good choice, since the Hrange isn't guaranteed to include 0.

Reply:
Oh -- actually you shouldn't even trigger this if the Hmark is None, since it's in an 'if' block where Hmark is not None? Is the summary stat not being calculated correctly?

Reply:
Ok, so the proper fix I think is to change the summary metric so that if Hmark is outside the range of computed values, it can extrapolate. (it's currently doing linear interpolation within the range if it doesn't match, so this should be a reasonable fix too).